src/plugins/mix.ts

```diff
@@ -21,6 +38,26 @@ const mixPlugin: Plugin = (ColordClass): void => {
     const mixture = mix(this.toRgb(), instance2.toRgb(), ratio);
     return new ColordClass(mixture);
   };
+
+  ColordClass.prototype._mixPalette = function (color, colors = 5) {
+    const palette = [];
+    for (let ratio = 0; ratio <= 1; ratio += 1 / colors) {
+      palette.push(this.mix(color, ratio));
+    }
+    return palette;
+  };
+
+  ColordClass.prototype.tints = function (colors) {
+    return this._mixPalette("#ffffff", colors);
+  };
+
+  ColordClass.prototype.shades = function (colors) {
+    return this._mixPalette("#000000", colors);
```

Comment:
To reduce a bundle size a bit:
```suggestion
    return this._mixPalette("#000", colors);
```